00-RELEASENOTES

```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
 ================================================================================
+Valkey 7.2.6 GA  -  Released Tue 30 July 2024
+================================================================================
+
+Upgrade urgency MODERATE: Program an upgrade of the server, but it's not urgent.
```

Comment:
```suggestion
Upgrade urgency Medium: This release fixes an incompatibility issue with modules compiled for Redis. For other users, it primarily is fixing uncommon bugs.
```
Normally we add more context about the upgrade here. I think the biggest thing is the module compatibility, everything else is just a minor improvement.